packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -265,22 +265,21 @@ const PRESETS = [
             type: TEMPLATE_SOURCE_BUNDLE,
             id: 'chakra-storefront'
         },
-        getQuestions: () =>
-            createRetailReactAppQuestions({
-                'project.hybrid': false,
-                'project.name': 'demo-storefront',
-                'project.commerce.instanceUrl': 'https://zzte-053.dx.commercecloud.salesforce.com',
-                'project.commerce.clientId': '1d763261-6522-4913-9d52-5d947d3b94c4',
-                'project.commerce.siteId': 'RefArch',
-                'project.commerce.organizationId': 'f_ecom_zzte_053',
-                'project.commerce.shortCode': 'kv7kzm78',
-                'project.commerce.isSlasPrivate': false,
-                'project.einstein.clientId': '1ea06c6e-c936-4324-bcf0-fada93f83bb1',
-                'project.einstein.siteId': 'aaij-MobileFirst',
-                'project.dataCloud.appSourceId': 'fb81edab-24c6-4b40-8684-b67334dfdf32',
-                'project.dataCloud.tenantId': 'mmyw8zrxhfsg09lfmzrd1zjqmg',
-                'project.demo.enableDemoSettings': false
-            }),
+        answers: {
+            'project.hybrid': false,
+            'project.name': 'demo-storefront',
+            'project.commerce.instanceUrl': 'https://production-sitegenesis-dw.demandware.net',
```

Comment:
In the `develop` branch, this preset no longer uses the zzte-053 instance anymore. So I've made sure it's up to date.